src/main/java/fi/dy/masa/litematica/schematic/SchematicaSchematic.java

```diff
@@ -686,9 +681,12 @@ private void readEntitiesFromNBT(NbtCompound nbt)
         this.entities.clear();
         NbtList tagList = nbt.getList("Entities", Constants.NBT.TAG_COMPOUND);
 
+        Litematica.logger.warn("SchematicaSchematic: executing Vanilla DataFixer for Entities DataVersion 1139 -> {}", LitematicaSchematic.MINECRAFT_DATA_VERSION);
+
         for (int i = 0; i < tagList.size(); ++i)
         {
-            this.entities.add(tagList.getCompound(i));
+            // Throw this data to the Data Fixer gods from Version MINECRAFT_DEFAULT_DATA_VERSION
+            this.entities.add(SchematicConversionMaps.updateEntity(tagList.getCompound(i), LitematicaSchematic.MINECRAFT_DEFAULT_DATA_VERSION));
```

Comment:
Attempts to apply the DATA FIXERS to Schematica .schematic files.
No clue if this works, because I don't have any of these files to test with.